misc/examples/src/main/scala/org/w3/banana/examples/SPARQLExample.scala

```diff
@@ -38,13 +40,14 @@ trait SPARQLExample extends SPARQLExampleDependencies { self =>
     /* creates a Sparql Select query */
 
     val query = parseSelect("""
-PREFIX ont: <http://dbpedia.org/ontology/>
-SELECT DISTINCT ?language WHERE {
- ?language a ont:ProgrammingLanguage .
- ?language ont:influencedBy ?other .
- ?other ont:influencedBy ?language .
-} LIMIT 100
-""").get
+                                PREFIX ont: <http://dbpedia.org/ontology/>
```

Comment:
the format was wrong before, but still not quite correct - see https://github.com/banana-rdf/banana-rdf/blob/series/0.8.x/rdf-test-suite/jvm/src/main/scala/org/w3/banana/SparqlUpdateEngineTest.scala#L30 for an example (note .stripMargin at end)